Documentation/Orchard-dependencies-and-libraries.markdown

```diff
@@ -43,7 +43,7 @@ Flexible HTML parsing and querying.
 ## Html5shim
 Provides Html 5 helpers.
 
-## jQuery & jQueryUI, jQuery ui.timepickr, jQuery utils, jQuery ScrollTo
+## jQuery & jQueryUI, jQuery ui.timepicker, jQuery utils, jQuery ScrollTo
```

Comment:
this one has been fixed a few times but the library was actually called [ui.timepickr](https://code.google.com/p/jquery-utils/wiki/UiTimepickr)